### ISSUE TITLE
Support reboots

### ISSUE THREAD

**Issue by akrherz** [matteocorti/check_tcptraffic#3]
Thanks for your efforts on this nagios plugin.  The issue I have is getting bad statistics when the server is rebooted.  With this plugin prior to your efforts, it would emit negative numbers :(  With your updates, it appears it assumes that the counters rolled over?  Is there a way to make this plugin survive reboots or should I be scripting the removal of the temp files on reboot?  thanks!

```
# cat check_tcptraffic_status-eth1root.pre_reboot 
1470670038 5113780130 281899415
# cat check_tcptraffic_status-eth1root
1470670147 523524513 5896992
# check_tcptraffic -i eth1 -s 1000  -c 500000000 -w 400000000
TCPTRAFFIC OK - eth1 34162170 bytes/s | TOTAL=34162170B;400000000;500000000 IN=-2709067B;; OUT=36871237B;; TIME=109B;;
```

**Comment by matteocorti** [comment]
It is indeed a bug, I will have to detect if the machine was rebooted

**Comment by akrherz** [comment]
Thanks @matteocorti for the response.  Perhaps checking `/proc/uptime` to see if the first value (uptime seconds) is smaller than the time delta since the last check_tcptraffic dump?

**Comment by matteocorti** [comment]
Hi

I just checked. The times are computed using the `time` function which uses the time since the epoch. This should not be affected by a reboot. Could you please check if the system time and date are correct after a reboot?

Matteo

**Comment by akrherz** [comment]
Hello, the issue is not the timestamps, the issue is that the byte counts are lower...

**Comment by matteocorti** [comment]
Makes sense. I'll fix it.

**Comment by matteocorti** [comment]
Fixed in 2.2.6
